.claude/skills/web-search-agent-evals/SKILL.md

```
@@ -70,58 +70,113 @@ cat data/comparisons/runs/2026-01-24/all-weighted.json | jq '.headToHead.pairwis
-|------|---------|--------|----------|
-| `test.jsonl` | 5 | `<web-search>` | `SEARCH_PROVIDER=builtin` |
-| `test-you.jsonl` | 5 | `<web-search mcp-server="ydc-server">` | `SEARCH_PROVIDER=you` |
-| `full.jsonl` | 151 | `<web-search>` | `SEARCH_PROVIDER=builtin` |
-| `full-you.jsonl` | 151 | `<web-search mcp-server="ydc-server">` | `SEARCH_PROVIDER=you` |
+**Unified prompt format:** All prompts use "Use web search to find:\n<query>" regardless of mode (builtin or MCP).
 
-**You.com MCP format is 39-45% faster** than builtin due to explicit server specification.
+Prompts are organized by dataset type, with each in its own directory:
+
+| File | Prompts | Metadata | Use With |
+|------|---------|----------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)